i18n/en.toml

```diff
@@ -174,47 +173,47 @@ setTargetTime = "none"
 description = "When should the vehicle be charged to {targetSoC}%?"
 today = "today"
 tomorrow = "tomorrow"
-targetIsInThePast = "The chosen time is in the past."
+targetIsInThePast = "Pick a time in the future, Marty."
```

Comment:
For example my parents would not get Marty reference..

Reply:
> For example my parents would not get Marty reference..

I think it's done on a way that's optional. Mine wouldn't get it as well.